content/applications/finance/fiscal_localizations/ecuador.rst

```diff
@@ -569,13 +569,93 @@ withholdings percentages apply.
    withholding of 70% in a new line with the same tax support, the system will allow you as long as
    the total of the bases matches the total from the :guilabel:`Vendor Bill`.
 
+eCommerce
+---------
+
+The :ref:`ATS Report module <ecuador/ats>` enables the following:
+
+- Choose the SRI Payment Method in each payment method's configuration.
+- Customers can manually input their identification type and identification number during the
+  eCommerce checkout process.
+- Automatically generate a valid electronic invoice for Ecuador at the end of the checkout process.
+
+Configuration
+~~~~~~~~~~~~~
+
+Website
+*******
+
+To generate an invoice after the checkout process, navigate to :menuselection:`Website -->
+Configuration --> Settings` and activate the :guilabel:`Automatic Invoice` option found under the
+:guilabel:`Invoicing` section.
+
+.. tip::
+   The invoice's email template can be modified from the :guilabel:`Invoice Email Template`.
+
+.. important::
+   The sales journal used for invoicing is the first in the sequence of priority in the
+   :guilabel:`Journal` menu.
+
+Payment providers
+*****************
+
+To activate the payment providers that should be used to capture eCommerce payments, navigate to
+:menuselection:`Website --> Configuration --> Payment Providers` section and then click on the
+:guilabel:`View other providers` button under the :guilabel:`Activate Payments` heading. From here,
+each payment provider can be configured by selecting a provider record. Refer to the :doc:`payment
+provider <../payment_providers>` documentation for more information.
+
+Payment methods
+~~~~~~~~~~~~~~~
+
+To activate one or more payment methods for a payment provider, click :guilabel:`→ Enable Payment
+Methods` within the :guilabel:`Configuration` tab of each provider.
+
+When configuring the payment method, it is **mandatory** to set the :guilabel:`SRI Payment Method`
+for each method. This field appears after you create and save the payment method for the first
+time.
+
+.. note::
+   Adding the :guilabel:`SRI Payment Method` is necessary to generate correctly the electronic
+   invoice from an eCommerce sale. Select a **payment method** to access its configuration menu and
+   the field.
+
+.. seealso::
+   :doc:`../payment_providers`
```

Comment:
I would label this "payment provider" as on lines 605-606 or the other way around, just to be consistent